src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.iOS.cs

```diff
@@ -424,9 +398,69 @@ UISwipeGestureRecognizer CreateSwipeRecognizer(SwipeDirection direction, Action<
 
 		[SupportedOSPlatform("ios13.0")]
 		[SupportedOSPlatform("maccatalyst13.0")]
-		CustomHoverGestureRecognizer CreatePointerRecognizer(Action<UIHoverGestureRecognizer> action)
+		List<UIGestureRecognizer?> CreatePointerRecognizer(WeakReference weakRecognizer, WeakReference weakEventTracker)
 		{
-			var result = new CustomHoverGestureRecognizer(action);
+			bool exited = false;
+
+			Action<UIGestureRecognizer> action = (pointerGesture) =>
+			{
+				if (weakRecognizer.Target is PointerGestureRecognizer pointerGestureRecognizer &&
+					weakEventTracker.Target is GesturePlatformManager eventTracker &&
+					eventTracker._handler?.VirtualView is View view &&
+					eventTracker._handler?.MauiContext?.GetPlatformWindow() is UIWindow window)
+				{
+					var originPoint = pointerGesture.LocationInView(eventTracker?.PlatformView);
+					var platformPointerArgs = new PlatformPointerEventArgs(pointerGesture.View, pointerGesture);
+
+					switch (pointerGesture.State)
+					{
+						case UIGestureRecognizerState.Began:
+							exited = false;
+							if (pointerGesture is UIHoverGestureRecognizer)
+								pointerGestureRecognizer.SendPointerEntered(view, (relativeTo) => CalculatePosition(relativeTo, originPoint, weakRecognizer, weakEventTracker), platformPointerArgs);
+							else
+								pointerGestureRecognizer.SendPointerPressed(view, (relativeTo) => CalculatePosition(relativeTo, originPoint, weakRecognizer, weakEventTracker), platformPointerArgs);
+							break;
+						case UIGestureRecognizerState.Changed:
+							if (exited)
+								break;
+
+							if (pointerGesture is UIHoverGestureRecognizer)
+								pointerGestureRecognizer.SendPointerMoved(view, (relativeTo) => CalculatePosition(relativeTo, originPoint, weakRecognizer, weakEventTracker), platformPointerArgs);
+							else
+							{
+								if (view.ToPlatform().Bounds.Contains(originPoint))
```

Comment:
should we change this to
eventTracker?.PlatformView

Just to make sure we're consistent?
 That way if `eventTracker?.PlatformView` changes to something else this code will all continue to work